4001-5000/LIT4020RegulaeHierarchia.xml

```diff
@@ -5,6 +5,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Regulae de Hierarchia ecclesiastica</title>
+            <title xml:id="t2" xml:lang="gez">ሥርዓተ፡ ቤተ፡ ክርስቲያን፡ ወማዕርጊሃ፡</title>
```

Comment:
The plural of ማዕርግ māʿǝrǝg is መዓርግ maʿārǝg, shouldn't we standardize it?

Reply:
thank you, donw